com.github.halfmexican.Mingle.metainfo.xml.in

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.github.halfmexican.Mingle.desktop</id>
+  <launchable type="desktop-id">com.github.halfmexican.Mingle.desktop</launchable>
+  <name>Mingle</name>
+  <summary>Combine emojis with Google's Emoji Kitchen</summary>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0-or-later</project_license>
+	<description>
+	  <p>Combine emojis with Google's Emoji Kitchen, and send them to friends. Written in Vala + GTK4</p>
+	</description>
+  <content_rating type="oars-1.1">
+    <content_attribute id="drugs-alcohol">mild</content_attribute>
+    <content_attribute id="drugs-tobacco">mild</content_attribute>
+    <content_attribute id="sex-themes">mild</content_attribute>
+    <content_attribute id="language-humor">mild</content_attribute>
```

Comment:
Missing screenshots, releases https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/

Reply:
also missing developer info (`developer_name` or `devloper` tags) and `url`.

Reply:
I've removed this file from the pr but added all the requested info and ran `appstreamcli validate`